.github/workflows/maestro_release.yaml

```diff
@@ -100,16 +100,23 @@ jobs:
           generate_release_notes: true
           name: ${{ needs.build.outputs.release_name }} – ${{ github.ref_name }}
 
-  open-followup-pr:
+  followup-pr:
     needs: [build, release]
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - name: Strike through used release name
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+
+      - name: Bump version in pyproject.toml and update README.md
```

Comment:
might make more sense to just say "run bump_version.py script" than listing in English what it does